manifests/controllerconfig.crd.yaml

```diff
@@ -277,12 +277,52 @@ spec:
                                         description: url is fully qualified URI with scheme https, that overrides the default generated endpoint for a client. This must be provided and cannot be empty.
                                         type: string
                                         pattern: ^https://
+                                  x-kubernetes-list-type: atomic
```

Comment:
controllerconfig CRD lives now in openshift/api , you will need to update https://github.com/openshift/api/tree/master/machineconfiguration/v1 and vendor latest changes here.

Reply:
@sinnykumari ControllerConfig in openshift/api has already been updated: https://github.com/openshift/api/blob/master/machineconfiguration/v1/0000_80_controllerconfig-TechPreviewNoUpgrade.crd.yaml#L1629 and vendored https://github.com/openshift/machine-config-operator/blob/master/vendor/github.com/openshift/api/machineconfiguration/v1/0000_80_controllerconfig-TechPreviewNoUpgrade.crd.yaml#L1064

I ran `make update` on my mco branch and that resulted in this manifest being updated. It appears that it is unnecessary. So, I will remove this from my commit.

Reply:
ah ok. @cdoern is this expected?

Reply:
@cdoern the same manifest is updated when I run `make update` on master. So, nothing specific to this PR. Removed this from my commit.

Reply:
This is expected. Make update brings in the new manifests and copies them where necessary. There is a hack script that lists all of the  CRDs we copy.